mycroft/skills/volume/vocab/en-us/IncreaseVolumeKeyword.voc

```diff
@@ -1,4 +1,7 @@
 rise volume
 raise volume
 boost volume
-increase volume
\ No newline at end of file
+increase volume
+i can't hear you
```

Comment:
Please get this removed from the PR, as it's unrelated. If you need help with your git-fu, someone in the community should be able to help you. 

Also, I'm not a fan of "i can't hear you" or "what did you say". The current implementation will simply increase the volume without repeating itself, and that doesn't seem particularly useful (and actually really annoying).